pcsx2-qt/MainWindow.cpp

```diff
@@ -712,14 +712,14 @@ void MainWindow::updateAdvancedSettingsVisibility()
 	m_ui.actionEnableVerboseLogging->setVisible(enabled);
 }
 
-void MainWindow::onToolsVideoCaptureToggled(bool checked)
+void MainWindow::onVideoCaptureToggled(bool checked)
 {
 	if (!s_vm_valid)
 		return;
 
 	// Reset the checked state, we'll get updated by the GS thread.
-	QSignalBlocker sb(m_ui.actionToolsVideoCapture);
-	m_ui.actionToolsVideoCapture->setChecked(!checked);
+	QSignalBlocker sb(m_ui.actionVideoCapture);
+	m_ui.actionVideoCapture->setChecked(false);
```

Comment:
If `onCaptureStopped` and `onCaptureStarted` slots set the checked state. Is it possible to remove these two lines here?

Reply:
Done